include/caf/events/force_power_down_event.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#ifndef _FORCE_POWER_DOWN_EVENT_H_
+#define _FORCE_POWER_DOWN_EVENT_H_
+
+#include <event_manager.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Simple event to force a quick power down
```

Comment:
We should add documentation for newly added events into generated rst files (I think that could be separate PR)

Reply:
How to generate it?

Reply:
Add proper doxygengroup to` nrf/include/caf_overview.rst`